gwcs/legacy_wcs.py

```diff
@@ -0,0 +1,82 @@
+# Module to wrap the old WCS in the gWCS framework
+
+from astropy.modeling import FittableModel, Parameter
+
+
+from astropy.wcs import WCS
+
+
+
+def _get_possible_wcs_parameters(wcs):
+    """
+    get possible Parameters
+    :param wcs:
+    :return:
+    """
+    return [item.replace('get_', '') for item in wcs.__class__.__dict__
+            if item.startswith('get_')]
+
+
+def read_fits_wcs(header=None):
+    """
+    Parameters
+    ----------
+        header : astropy.io.fits header object, string, dict-like, or None, optional
+            If *header* is not provided or None, the object will be
+            initialized to default values.
+    """
+    legacy_wcs = WCS(header)
+
+
+    possible_parameters = _get_possible_wcs_parameters(legacy_wcs.wcs)
+
+    param_dict = {}
+    class_dict = {}
+    for param in possible_parameters:
+        param_value = getattr(legacy_wcs.wcs, 'get_{0}'.format(param))()
```

Comment:
Most of the params are properties -- the `get_` functions are really just there to get a normalized version of pc/cd/crota.  I think instead we should just have a white list of keys that make sense as fittable and use those.

Or does it make sense to expose everything as parameters and mark them as not fittable?  (I say, being somewhat ignorant of `astropy.modeling` still...)  Cc: @embray